chart/flux/templates/helm-operator-deployment.yaml

```diff
@@ -67,9 +70,9 @@ spec:
         {{- end }}
         {{- end }}
         args:
-        - --git-url={{ .Values.git.url }}
-        - --git-branch={{ .Values.git.branch }}
-        - --git-charts-path={{ .Values.git.chartsPath }}
+        - --git-url={{ $gitURL }}
```

Comment:
If you're using a different repo, you almost certainly need to mount a different SSH key. Unlike fluxd, we don't try to generate one (or give people a way to ask for it). The best that can be done for now is for people to create a secret with a private key, install the public key at github (or wherever), and supply the secret name as a parameter.

Reply:
You can simply get around this by just reusing the same key as Flux uses with read only rights?

Reply:
github won't let you use the same SSH key for more than one repo.

Reply:
Oh, you're completely right. I totally forgot about this restriction.

Reply:
@squaremo I've added the option to specify a SSH key secret for helm-op